packages/plugins/geth/src/index.js

```diff
@@ -73,22 +78,30 @@ class Geth {
   }
 
   _getNodeState(err, version, cb) {
-    if (err) return cb({name: "Ethereum node not found", status: 'off'});
+    if (err) return cb({ name: "Ethereum node not found", status: 'off' });
```

Comment:
I know this hasn't been introduced in this PR, but just a comment for future improvement:

We're using `name` here to emit a status message. I think we should either introduce a `message` property, or, to make things language agnostic, just use the information propagated in `status` to decide what message should be shown.

Reply:
Agreed.